api/serializers.py

```diff
@@ -11,6 +11,16 @@ class Meta:
         fields = ('id', 'source', 'app_label', 'provider_doc_id', 'data', 'sha256', 'date_seen', 'date_harvested')
 
 
+class ProviderRegistrationSerializer(serializers.ModelSerializer):
+    status = serializers.IntegerField(read_only=True)
```

Comment:
There's a related comment in the frontend PR. Could you return this as the string variant?